### PR TITLE
Using 'get_key' instead of 'new_key' and added new copy method

### PR DESCRIPTION
**Pull request by crankycookie**

### CODE REVIEW

File: oz/aws_cdn/__init__.py [hunk 1]
```diff
@@ -157,3 +157,10 @@ def exists(self):
 
     def remove(self):
         self.key.delete()
+
+    def copy_to(self, new_path):
+        '''
+        Uses boto to copy the file to the new path instead of uploading another file to the new key
+        '''
+        bucket = self.key.bucket
+        bucket.copy_key(new_path, bucket.name, self.key.name)
```
Comment (ysimonson) [inline oz/aws_cdn/__init__.py]: Does this copy metadata/headers as well? If not we should add that.

**Comment by crankycookie** [comment]
@ysimonson boto's `copy` methods also copy the headers.

**Comment by ysimonson** [comment]
1) Does the s3 copy call also copy metadata/headers? If not we should add that.
2) Could you add tests?

**Comment by crankycookie** [comment]
1. Yep, it copies the headers and metadata too.
2. What kind of tests were you thinking? There are already some `copy` tests in place that the new changes reuse.

**Comment by ysimonson** [comment]
Oh okay, that works

**Comment by crankycookie** [comment]
@ysimonson can we get this merged into master?

**Comment by ysimonson** [comment]
Done

**Pull request merged**